src/components/link/link.tsx

```diff
@@ -123,13 +112,24 @@ const EuiLink = forwardRef<HTMLAnchorElement | HTMLButtonElement, EuiLinkProps>(
       </EuiScreenReaderOnly>
     );
 
+    const externalLinkIcon = (
+      <EuiIcon
+        aria-label={useEuiI18n('euiLink.external.ariaLabel', 'External link')}
+        size="s"
+        css={cssExternalLinkIconStyles}
+        type="popout"
+      />
+    );
+
     if (href === undefined || !isHrefValid) {
       const buttonProps = {
-        className: classNames(
-          'euiLink',
-          disabled ? 'euiLink-disabled' : colorsToClassNameMap[color],
-          className
-        ),
+        className: classNames('euiLink', className),
+        css: [
+          cssStyles,
+          styles.button,
+          !disabled ? styles[color] : [],
+          disabled ? [styles.disabled] : [],
```

Comment:
Nit, let's use a ternary here and combine these
```suggestion
          disabled ? [styles.disabled] : styles[color],
```

Reply:
Haha, great catch. My brain clearly wasn't working when I wrote that originally!